packages/tldraw/package.json

```diff
@@ -58,6 +58,7 @@
 		"canvas-size": "^1.2.6",
 		"classnames": "^2.3.2",
 		"hotkeys-js": "^3.11.2",
+		"lodash.throttle": "^4.1.1",
```

Comment:
Instead of removing `throttle` from `@tldraw/utils` and adding this in the `tldraw` package, could we add this dependency to `@tldraw/utils` and re-export it from there? I'd like there to be one sort of "blessed" way to do throttling and `@tldraw/utils` feels like a good way to signal that.